packages/docker-bin/docker-bin.pacscript

```diff
@@ -0,0 +1,29 @@
+name="docker-bin"
+gives="docker"
+replace="${gives}"
+repology=("project: ${gives}")
+version="23.0.1"
+depends="procps git iptables"
+description="Docker Engine Static Binaries"
+
+if [ $(dpkg --print-architecture) = arm64 ]; then
```

Comment:
```suggestion
if [[ $CARCH == "amd64" ]]; then
```

Reply:
Do you mean $CARCH == arm64 or $CARCH !== amd64

Reply:
If `$CARCH == 'amd64'`

Reply:
why tho